packages/core/src/editor/Block.css

```diff
@@ -174,9 +174,10 @@ NESTED BLOCKS
 }
 
 /* Ordered */
-.bn-block-content[data-content-type="numberedListItem"] {
+.bn-block-content[data-content-type="numberedListItem"]::before {
   display: flex;
-  gap: 0.5em;
+  justify-content: center;
+  width: 25px;
```

Comment:
I don't think this (or the others) should be a fixed width, since the contents are actually variable

<img width="197" alt="image" src="https://github.com/user-attachments/assets/feeb26bd-0edb-4e33-ab39-6593a91d02cf" />